crates/bevy_audio/src/audio_output.rs

```diff
@@ -79,7 +80,11 @@ where
             if let Some(audio_source) = audio_sources.get(&config.source_handle) {
                 if let Some(sink) = self.play_source(audio_source, config.settings.repeat) {
                     sink.set_speed(config.settings.speed);
-                    sink.set_volume(config.settings.volume);
+                    if config.settings.absolute_volume {
```

Comment:
Could be replaced with something like this if it's preferred.
```rs
sink.set_volume(config.settings.volume * if config.settings.absolute_volume { 1.0 } else { global_volume.volume });
```

Reply:
I think this is a bit more clear.

Reply:
The current solution or the proposed one?

Reply:
Current solution.